README.md

```diff
@@ -1,7 +1,7 @@
 # Rossby
 
-[![Build Status](https://travis-ci.com/kansaslabs/rossby.svg?branch=master)](https://travis-ci.com/kansaslabs/rossby)
-[![codecov](https://codecov.io/gh/kansaslabs/rossby/branch/master/graph/badge.svg)](https://codecov.io/gh/kansaslabs/rossby)
+[![Build Status](https://travis-ci.com/kansaslabs/rossby.svg?branch=develop)](https://travis-ci.com/kansaslabs/rossby)
+[![codecov](https://codecov.io/gh/kansaslabs/rossby/branch/develop/graph/badge.svg)](https://codecov.io/gh/kansaslabs/rossby)
```

Comment:
This is part of making the develop branch the primary branch of Rossby -- similar to what we did in Yellowbrick.